code/game/objects/items/mop.dm

```diff
@@ -21,7 +21,7 @@
 
 /obj/item/mop/Initialize(mapload)
 	. = ..()
-	create_reagents(mopcap)
+	create_reagents(mopcap, REFILLABLE)
```

Comment:
Reason for this?

Reply:
In order to allow sinks to refill the gun, I changed sinks to refill items based on whether or not they have a refillable reagent holder, so I also made mops refillable now that they don't need a special check anymore.